### PR TITLE
added docs for ios

### PR DESCRIPTION
**Pull request by joalux2**
New documentation for iOS.

### CODE REVIEW

File: docs/getting-started/ios/index.md [hunk 1]
```diff
@@ -3,3 +3,56 @@
...
+
+Creating component:
+
+    Button:
+
+        Warp.Button(type: <#T##Warp.ButtonType#>, title: <#T##String#>, imageSystemName: <#T##String?#>, action: <#T##() -> Void#>, size: <#T##Warp.ButtonSize#>, isEnabled: <#T##Bool#>, fullWidth: <#T##Bool#>, colorProvider: <#T##ColorProvider#>)
+
+        Warp.Button.create(for: .primary, title: "Warp", icon: Image(systemName: "plus")) {
+            print("Warp button")
+       }
```
Comment (siljec) [inline docs/getting-started/ios/index.md]: Can you wrap all the code examples with code formatting, i .e. like you are doing here: 
image

Comment (siljec) [reply docs/getting-started/ios/index.md]: Or wait... I think the examples of how to use the button component belongs in the "component" section, i.e. [here](https://warp-ds.github.io/tech-docs/components/buttons/) for button. If you scroll down you can choose between web frameworks (React, Vue, Elements) and Android. We want to add iOS here as well. But that can be done in a separate task.

File: docs/getting-started/ios/index.md [hunk 2]
```diff
@@ -3,3 +3,56 @@
 
 This page describes how to get started building an application with Warp components.
 
+## 1. Integrate Warp
+
+A guide on how to integrate Warp into your project.
+
+### Swift Package Manager
+
```
Comment (siljec) [inline docs/getting-started/ios/index.md:14]: Do we need to give more explanations here? Is this the only thing a user needs to do in order to integrate Warp? 

Can we add:
```suggestion
To get started, you need to install the `warp-ds/warp-ios` package

```

**Comment by siljec** [comment]
How do the user apply the theme? I.e. choose between the Finn brand, or Tori brand?

File: docs/getting-started/ios/index.md [hunk 3]
```diff
@@ -1,5 +1,75 @@
...
+var brand: String = "" {
+    didSet {
+        if brand == "Tori" {
+             iconImageView.tintColor = UIColor(red: 247/255, green: 81/255, blue: 89/255, alpha: 1)
+
+        }
+        else if brand == "Finn" {
+           iconImageView.tintColor = UIColor(red: 6/255, green: 190/255, blue: 251/255, alpha: 1)
+
+        }
```
Comment (siljec) [inline docs/getting-started/ios/index.md]: Hmm is it not a better way to switch the branding, than to use if else sentences and hardcoding the color?

File: docs/api-table.js [hunk 4]
```diff
@@ -1661,3 +1661,127 @@ export const android = {
...
+      ],
+      [
+        'title',
+        'String?',
+        'nil',
+        'The title to display above the textfield',
+      ],
+      [
+        'additionalInformation',
+        'Srring?',
```
Comment (felicia-haggqvist) [inline docs/api-table.js]: Assuming this is a typo: `Srring?` and should instead be `String?`? :)

**Review by felicia-haggqvist — COMMENTED** [review]
Also commented on [this post](https://sch-chat.slack.com/archives/C050N3NQ204/p1705407577606059)  in our #nmp-warp-devs slack-channel, regarding `iOS` getting capitalized in the `Frameworks` tabs content, but I'll post my answer here as well :) :  You can inspect the `TabsContent.vue` file. There it seems to have a `formatTabTitle` function that capitalizes the titles in the tabs content. Maybe you could refactor it and not capitalize it if it is `iOS` ? So maybe something like this:
```
const formatTabTitle = (tab) => {
 if (tab !== 'iOS') {
  const capitalizedTab = tab.charAt(0).toUpperCase() + tab.slice(1);
  return capitalizedTab?.replaceAll('_', ' ');
 } else {
  return tab
 }
}; 
```

File: docs/api-table.js (hunk 4 above)
Comment (mohsen-vend) [reply docs/api-table.js]: yes, thank you for reading it line by line 😅

**Comment by felicia-haggqvist** [comment]
I also just noticed that your branch is 17 commits behind master. Could maybe be good to update your branch so that it has the latest changes and see that nothing has broken for you? :) We have, for example, updated the @warp-ds/css, @warp-dcs/icons and @warp-ds/vue dependencies.

File: docs/.vitepress/TabsContent.vue [hunk 5]
```diff
@@ -6,8 +6,12 @@ const slots = useSlots();
 const tabs = Object.keys(slots);
 
 const formatTabTitle = (tab) => {
-  const capitalizedTab = tab.charAt(0).toUpperCase() + tab.slice(1);
-  return capitalizedTab?.replaceAll('_', ' ');
+  if (tab == 'iOS') {
```
Comment (felicia-haggqvist) [inline docs/.vitepress/TabsContent.vue]:
```suggestion
  if (tab === 'iOS') {
```

**Comment by mohsen-vend** [comment]
> I also just noticed that your branch is 17 commits behind master. Could maybe be good to update your branch so that it has the latest changes and see that nothing has broken for you? :) We have, for example, updated the @warp-ds/css, @warp-dcs/icons and @warp-ds/vue dependencies.

Done

**Review by felicia-haggqvist — APPROVED** [review]
🎉

**Pull request merged**